package.json

```diff
@@ -1,14 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp test"
+    "build": "spago build",
+    "test": "spago build && purescm bundle-app --main Test.Main && scheme --program output/main"
```

Comment:
Just a thought: we might want to add a `test` command to `purescm`, that does the bundle-app + call to scheme